conf/machine/skipjack.conf

```diff
@@ -12,4 +12,4 @@ PREFERRED_VERSION_android = "oreo"
 PREFERRED_PROVIDER_virtual/kernel = "linux-skipjack"
 PREFERRED_VERSION_linux = "3.10+pie"
 
-IMAGE_INSTALL += "android-tools android-system underclock android-system-partition msm-fb-refresher asteroid-compass"
+IMAGE_INSTALL += "android-tools android-system underclock android-system-partition msm-fb-refresher asteroid-compass bluebinder libgbinder libglibutil"
```

Comment:
You shouldn't need to install `libgbinder` and `libglibutil` as they are only build time dependencies for `bluebinder`. Only `bluebinder` should be good enough.

Reply:
Done!